app/models/user.rb

```diff
@@ -28,8 +27,17 @@ def third_party_authenticated?(request)
     request.env.key?('omniauth.auth') ? true : false
   end
 
+  def formatted_mobile
+    mobile && mobile.phony_formatted(format: :international, normalize: :US, spaces: ' ')
+  end
+
+  def mobile_confirm(new_mobile)
+    SmsSenderNumberChangeJob.perform_later(mobile) if mobile.present?
+    update(mobile: new_mobile, mobile_confirmed_at: Time.current)
+  end
+
```

Comment:
This method does not belong in the User model. Confirming a mobile number is not a User concern. Models should be as lean as possible and should ideally only deal with persistence. Business logic should be moved to controllers or specialized classes. In this case, I think we can just use this method in the controller, much like the `send_number_change_sms_if_needed` that we used to have.